prac_05/wimbledon.py

```diff
@@ -0,0 +1,42 @@
+"""
+CP1404/CP5632 Practical
+Wimbledon winners
+"""
+
+
+def main():
+    winners = []
+    winner_to_nationality = {}
```

Comment:
Is this dictionary containing winners and their nation? Or is it is the count of how many times they have won?